ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneIndexService.scala

```diff
@@ -34,65 +31,19 @@ object StandaloneIndexService {
   )(implicit
       materializer: Materializer,
       loggingContext: LoggingContext,
-  ): ResourceOwner[IndexService] = {
-    val valueEnricher = new ValueEnricher(engine)
-
-    def preloadPackages(packageContainer: InMemoryPackageStore): Unit = {
-      for {
-        (pkgId, _) <- packageContainer.listLfPackagesSync()
-        pkg <- packageContainer.getLfPackageSync(pkgId)
-      } {
-        engine
-          .preloadPackage(pkgId, pkg)
-          .consume(
-            { _ =>
-              sys.error("Unexpected request of contract")
-            },
-            packageContainer.getLfPackageSync,
-            { _ =>
-              sys.error("Unexpected request of contract key")
-            },
-          )
-        ()
-      }
-    }
```

Comment:
No preloading anymore?

Reply:
In the old code `IndexConfiguration.archiveFiles` is not used, so I removed the parameter and all the code associated with it